presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -5,6 +5,7 @@ services:
     image: 'teradatalabs/centos6-java8-oracle:${DOCKER_IMAGES_VERSION}'
     volumes:
       - ../..:/docker/volumes/conf
+      - ../../target:/docker/volumes/logs
```

Comment:
I don't think so, we probably would need plug in s/w around here:
```
function run_product_tests() {
  local REPORT_DIR="${PRODUCT_TESTS_ROOT}/target/test-reports"
  rm -rf "${REPORT_DIR}"
  mkdir -p "${REPORT_DIR}"
```
But I think additional 3 lines of code are not worth the `../../target/logs/` comment, let's drop it :)